app/templates/components/user-menu.hbs

```diff
@@ -1,23 +1,25 @@
 {{#if session.isOpening}}
   <li class="dropdown disabled signing-in">Signing in...</li>
 {{else}}
-  <span class="user-avatar">
-    {{#if session.isAuthenticated}}
-      <img src="{{session.currentUser.avatarUrl32}}" />
-    {{/if}}
-  </span>
-  <li class="dropdown">
-     <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-       {{userName}} <b class="caret"></b>
-     </a>
-     <ul class="dropdown-menu dropdown-menu-right">
-       {{#if session.isAuthenticated}}
-         <li><a {{action 'signOut'}} class="test-sign-out">Sign out</a></li>
-       {{else}}
-         <li><a {{action 'signInViaGithub'}} class="test-sign-in">Sign in</a></li>
-       {{/if}}
-     </ul>
-  </li>
+  {{#if session.isAuthenticated}}
+    <span class="user-avatar">
+        <img src="{{session.currentUser.avatarUrl32}}" />
+    </span>
+
+    <li class="dropdown">
+      <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+        {{userName}} <b class="caret"></b>
+      </a>
+      <ul class="dropdown-menu dropdown-menu-right">
+        <li><a {{action 'signOut'}} class="test-sign-out">Sign out</a></li>
+      </ul>
+    </li>
+  {{else}}
+    <li><a {{action 'signInViaGithub'}} class="test-sign-in sign-in">Sign in with Github</a></li>
```

Comment:
Can you make this just Sign in? It takes up too much space on mobile and small screens.